.github/workflows/ci.yaml

```diff
@@ -100,7 +100,15 @@ jobs:
           go version
           go get -u golang.org/x/lint/golint
           go get github.com/stretchr/testify/assert
-
+      
+      - name: Running apisonator
```

Comment:
CI fails because you are running apisonatar from here and from the integration tests as well.

Reply:
Done